x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -384,6 +401,15 @@ const UnstyledProcessEventDot = React.memo(
             color={colorMap.descriptionText}
             isDisplaying={isShowingDescriptionText}
           >
+            {isOrigin && (
+              <>
```

Comment:
Maybe put the 'analyzed event' text, the dot, and the `descriptionText` into a single formatted message. This will allow translators to, for example, swap out the dot with something locale specific, or reverse the order.

Reply:
Yea, makes sense. I've updated it